README.rst

```diff
@@ -537,16 +537,47 @@ the run step:
           namespaced.file.alias2: "/path/to/readwrite/file/or/dir:rw"
           build/dir/file: "/path/to/build/dir/file"
 
-        # A map specifying cache directories that should be mounted inside the
-        # container. The cache directories are maintained between builds and can
-        # be used to store files, such as downloaded dependencies, to speed up
-        # builds. Caches are shared within a build configuration, meaning that
-        # caches with the same name are shared between steps. Caches should be
-        # treated as ephemeral and should only store items that can be
-        # obtained/generated by subsequent builds.
+        # A map specifying cache directories that stored as archive files on the
+        # host system as named <local cache key> and extracted as directory in
+        # the container name <docker path>. The cache directories are maintained
+        # between builds and can be used to store files, such as downloaded
+        # dependencies, to speed up builds. Caches are shared within a build
+        # configuration, meaning that caches with the same name are shared between
+        # steps. Caches should be treated as ephemeral and should only store items
```

Comment:
I would change the "Caches are shared" to "Caches can be shared between any builds or projects on the system as the names are not prefixed with any project-specific information."

Reply:
- [ ] Update comment change the "Caches are shared" to "Caches can be shared between any builds or projects on the system as the names are not prefixed with any project-specific information."